pkg/scan/apk.go

```diff
@@ -139,11 +139,16 @@ func NewScanner(opts Options) (*Scanner, error) {
 		dbDestDir = DefaultGrypeDBDir
 	}
 
+	grypeMaxAllowedBuiltAge, err := time.ParseDuration(os.Getenv("GRYPE_DB_MAX_ALLOWED_BUILT_AGE"))
+	if err != nil {
+		grypeMaxAllowedBuiltAge = time.Duration(24)
+	}
+
 	installCfg := installation.Config{
 		DBRootDir:               dbDestDir,
 		ValidateChecksum:        true,
 		ValidateAge:             !opts.DisableDatabaseAgeValidation,
-		MaxAllowedBuiltAge:      24 * time.Hour,
+		MaxAllowedBuiltAge:      grypeMaxAllowedBuiltAge * time.Hour,
```

Comment:
to fix `Multiplication of durations:` here, it think we should do:

```go
MaxAllowedBuiltAge:  time.Duration(grypeMaxAllowedBuiltAge) * time.Hour,
```

Reply:
`grypeMaxAllowedBuiltAge` already shared the type. I was multiplying two durations which is wrong. Should be sorted now. Thank you for taking a look!